src/core/simulation/circuit.rs

```diff
@@ -28,7 +28,7 @@ impl Circuit {
     pub fn tick(&self) {
         for clock_idx in self.clock_generators.iter() {
             let clock = self.get_component(*clock_idx);
-            
+
```

Comment:
*excuse me wtf*

Reply:
Formatter got rid of unnecessary tabs and spaces)